### PR TITLE
fix(personal-representative): Updated builder for personal-rep services

### PR DESCRIPTION
**Pull request by FridrikMProgramm**
Updating builder for personal-rep services. Adding needed esbuild files as well
Getting tslint missing errors with current builder.

## Screenshots / Gifs
image

## Checklist:

- [X ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: workspace.json [hunk 1]
```diff
@@ -8008,7 +8008,7 @@
       "prefix": "personal-representative",
       "architect": {
         "build": {
-          "builder": "@nrwl/node:build",
+          "builder": "@irdnis/esbuildnx:build",
```
Comment (saevarma) [inline workspace.json:8011]: For both services you need to add `esbuild.json`  file to list external dependencies.
You can read more about it [here](https://esbuild.github.io/api/#external).

You can find examples of `esbuild.json` from other services in the repo.
Like [this one](https://github.com/island-is/island.is/blob/main/apps/services/auth-api/esbuild.json)

**Review by saevarma — DISMISSED** [review]

File: workspace.json (hunk 1 above)
Comment (FridrikMProgramm) [reply workspace.json:8011]: Just added :)

**Review by darrikonn — APPROVED** [review]
LGTM

**Review by fjandakornid — APPROVED** [review]
LGTM

**Pull request merged**